.github/workflows/integration.yml

```diff
@@ -26,6 +28,9 @@ jobs:
         echo "::set-output name=address::https://fleet-test-$UUID.fleetuem.com"
   
   run-server:
+    permissions:
+      actions: read  # for dawidd6/action-download-artifact to query and download artifacts
+      pull-requests: read  # for dawidd6/action-download-artifact to query commit hash
```

Comment:
Are these necessary? It seems like the job has been functioning with the `contents: read` permission only: https://github.com/fleetdm/fleet/actions/workflows/integration.yml

I wonder if we are best off just setting this entire workflow to `contents: read` rather than trying to do a per job permission?

Reply:
Yeah you are totally right. Also, as pointed in another comment, content:none is not really gaining us anything with a public repo anyway.